### ISSUE TITLE
Stack formatting in REPL

### ISSUE THREAD

**Issue by sharpvik** [ForthScriptLang/forthscript#26]
Loving the language so far. Very intuitive and clear. However, as the stack grows bigger, the `[]#` shifts right. Consider displaying it in the following way:

```
[True False 42 "Hello, World!"]
↪
```

Then, the input line is separate from the stack.

**Comment by sharpvik** [comment]
> Apologies, I only saw this issue now.

No worries at all, it's your project after all. You don't even have to agree with my opinion at all.

I just thought it might be a nice (but also super quick and easy) improvement.

I'm using the suggested layout in my default shell because when you cd deep into the folder tree, the path gets long and there might not be enough space for the actual command. So I changed it to this.

When I saw your REPL, I thought to myself that you have the same exact problem, so I decided to offer a solution.

I wish you the best of luck with ForthScript in the future :)
